### PR TITLE
{lib}[GCCcore/12.3.0] makefun 1.15.2

### PR DESCRIPTION
**Pull request by ThomasHoffmann77**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/a9f2be19dd8ae8a6f2536725770b90d9 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3117.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/f5be2aeb94ba129eb220bd8577bff00b for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/m/makefun/makefun-1.15.2-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    (name, version, {
+        'checksums': ['16f2a2b34d9ee0c2b578c960a1808c974e2822cf79f6e9b9c455aace10882d45'],
+    }),
+]
+
+sanity_pip_check = True
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/'],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/m/makefun/makefun-1.15.2-GCCcore-12.3.0.eb:34]:
```suggestion
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3117.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/800e8f5e6051cdee0c1aae47766a4b0f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ThomasHoffmann77!

**Pull request merged**